Client/Help/src/app/components/content-messages-gallery/content-messages-gallery.component.html

```diff
@@ -0,0 +1,16 @@
+<main class="flex flex-col justify-around items-center">
+    <section *ngFor="let profile of arrayOfProfiles; index as i" [ngClass]="{'bg-blue-300': i===0, 'bg-green-300': i%2=== 0, 'bg-yellow-300': i%2 !== 0 }" class="flex flex-row bg-blue-300 w-11/12 h-24 mt-4 pt-2">
+        <article class="ml-1 mr-4">
+            <fa-icon [icon]="profilePic" class="text-4xl"></fa-icon>
+        </article>
+        <article class="w-full">
+            <section class="flex justify-between mr-2">
+                <p>{{profile.name}}</p>
+                <p>{{profile.lastMessageTime}}</p>
```

Comment:
Why is it a messageTime, but it is an integer instead of a timestamp? I'm confused by what this is meant to represent.

Reply:
Most likely bad naming on my part. The idea was in the future I know we going to have to do calculations with a timestamp to show how long ago the message was. But for dummy data purposes I used a number.

I'm going to rename it lastMessageTimeStamp. In the future we need to do the necessary calculations to output the correct time unites (minutes, hours, days).